efficientdet/dataloader.py

```diff
@@ -48,28 +48,11 @@ def __init__(self, image, output_size):
     self._crop_offset_y = tf.constant(0)
     self._crop_offset_x = tf.constant(0)
 
-  @property
-  def image(self):
-    return self._image
-
-  @image.setter
-  def image(self, image):
-    self._image = image
```

Comment:
Could keep the api since I use them in other models.

Reply:
Done